openwisp_monitoring/db/backends/influxdb/creation.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from .. import TIMESERIES_DB
+
+logger = logging.getLogger(__name__)
+
+
+class DatabaseCreation(object):
```

Comment:
Not sure about this class. Have you taken inspiration from somewhere else?

Reply:
From how Django does it https://github.com/django/django/blob/master/django/db/backends/sqlite3/creation.py

Reply:
If you want to follow the same pattern, you have to subclass `django.db.backends.base.creation.BaseDatabaseCreation`